test/celeritas/optical/Cherenkov.test.cc

```diff
@@ -483,6 +498,54 @@ TEST_F(CherenkovTest, generator)
         }
     }
 }
+class CherenkovAirTest : public CherenkovTest
+{
+  public:
+    ImportOpticalProperty build_import_property() const final
+    {
+        ImportOpticalProperty prop;
+        std::vector<double> wl = {1.29, 1.20, 1.10, 1.00, 0.90, 0.82};
+        prop.refractive_index.x.reserve(wl.size());
+        for (double w : wl)
+        {
+            prop.refractive_index.x.push_back(um_to_mev(w));
+        }
+        prop.refractive_index.y = {1.30, 1.305, 1.31, 1.31, 1.320, 1.330};
+        return prop;
+    };
+};
+
+TEST_F(CherenkovAirTest, dndx)
+{
+    EXPECT_SOFT_NEAR(369.81e6,
+                     constants::alpha_fine_structure * units::Mev::value()
+                         * units::centimeter
+                         / (constants::hbar_planck * constants::c_light),
+                     1e-6);
+
+    optical::MaterialView mat_view{material->host_ref(), material_id};
+    CherenkovDndxCalculator calc_dndx(
+        mat_view,
+        params->host_ref(),
+        this->particle_params()->get(ParticleId{0}).charge());
+
+    std::vector<real_type> betas{
+        real_type{1.0 / 1.2}, real_type{1.0 / 1.3}, real_type{1.0 / 1.4}};
```

Comment:
We should test one of the constant values, it should work now:
```suggestion
        real_type{1.0 / 1.2}, real_type{1.0 / 1.3}, real_type{1.0 / 1.31}, real_type{1.0 / 1.4}};
```

Reply:
Updated. Thank you 😊